spec/features/users_spec.rb

```diff
@@ -1,7 +1,28 @@
 require 'rails_helper'
 
 RSpec.feature "Users", type: :feature do
-  
+  context 'create new user' do
+    before(:each) do
```

Comment:
You can skip `(:each)` it works for each by default
Btw, good idea to move that part to before block, 👍